enterprise/frontend/src/metabase-enterprise/metabot/index.tsx

```diff
@@ -34,13 +36,20 @@ if (hasPremiumFeature("metabot_v3")) {
             if (searchText) {
               submitInput(searchText);
             }
+            // HACK: if the user opens the command palette via the search button bar focus
+            // will be moved back to the search button bar if the metabot option is chosen
+            setTimeout(() => {
+              document.getElementById("metabot-chat-input")?.focus();
```

Comment:
We could leverage some of the Mantine [focus trapping ](https://v7.mantine.dev/core/focus-trap/)components here to maybe do stuff a bit cleaner.

Reply:
I'm not able to make FocusTrap work, can we look into this tomorrow?

Reply:
I'm not able to make this work. Let's look at this whenever you're back in office. I don't think it's so egregious we can't merge and fix later.

Reply:
Agreed 👍